bciers/apps/reporting/src/app/components/operations/OperationReviewFormData.tsx

```diff
@@ -13,13 +15,23 @@ export default async function OperationReviewFormData({
   const allActivities = await getAllActivities();
   const allRegulatedProducts = await getRegulatedProducts();
   const reportingYear = await getReportingYear();
+  const reportType = await getReportType(version_id);
+  const registrationPurpose = await getRegistrationPurpose(version_id);
+  const facilityReport = await getFacilityReport(version_id);
+
+  const registrationPurposeString =
+    registrationPurpose?.registration_purposes?.join(", ") || "";
```

Comment:
I believe this has been changed to just one purpose per operation now https://github.com/bcgov/cas-registration/pull/2460